### PR TITLE
Move Firefox perfstats results to browsertime.json

### PR DESCRIPTION
**Pull request by dpalmeiro**
Since the perfstats data is not large and it is mostly internal metric measurements, it seems a better fit to output the results in browsertime.json instead of separate files.

### CODE REVIEW

File: lib/core/engine/collector.js [hunk 1]
```diff
@@ -357,6 +357,14 @@ class Collector {
         });
       }
 
+      // Add Firefox perfStats if available
+      if (this.options.firefox && this.options.firefox.perfStats) {
+        if (!results.perfStats) {
+          results.perfStats = [];
```
Comment (soulgalore) [inline lib/core/engine/collector.js:363]: Hi @dpalmeiro sorry I've been offline for a week, I'm back now. I haven't tested the perfStats yet, but what about just rename the field _perfStats_ to something more descriptive? Is it internal browser performance statistics or what is collected?

**Comment by soulgalore** [comment]
Also @dpalmeiro do you have a link to documentation or a task about the flags in `--firefox.perfStatsParams.mask`?

**Comment by dpalmeiro** [comment]
@soulgalore The current flags in the mask are here:
https://searchfox.org/mozilla-central/source/tools/performance/PerfStats.h#24-33

File: lib/core/engine/collector.js (hunk 1 above)
Comment (dpalmeiro) [reply lib/core/engine/collector.js:363]: Sure, I'm open to any naming changes.  Maybe geckoPerfStats or something if that is more clear, or anything you may think sounds better.

Comment (soulgalore) [reply lib/core/engine/collector.js:363]: Yes, geckoPerfStats sounds get better, I'm just thinking so it do not collide with something else in the future. I can change it if you want, I wanna push the release later tonight.

Comment (dpalmeiro) [reply lib/core/engine/collector.js:363]: Sure, please go ahead. Thanks!

**Pull request merged**